StatusBar.qml

```diff
@@ -0,0 +1,59 @@
+import QtQuick 2.0
+
+Rectangle {
+    Text {
```

Comment:
Take a note that these two elements are mostly duplicated. Move it to another general component and reuse.